go/chat/sender.go

```diff
@@ -1372,6 +1376,11 @@ func (s *NonblockingSender) Send(ctx context.Context, convID chat1.ConversationI
 	if err != nil {
 		return nil, nil, err
 	}
+
+	// Add txIDs to client header, if relevant
+	bodyTxIDs := utils.GetMessageBodyTxIDs(msg.ClientHeader.MessageType, msg.MessageBody)
+	msg.ClientHeader.TxIDs = &bodyTxIDs
```

Comment:
hoping that all messages go through this nonblocking Send method and the blocking Send method, so that the TxIDs always get added

Reply:
not needed here, just in prepare